python/plugins/processing/algs/gdal/ExecuteSql.py

```diff
@@ -77,7 +77,7 @@ def name(self):
         return 'executesql'
 
     def displayName(self):
-        return self.tr('Execute SQL')
+        return self.tr('Execute SQL and returns a OGR SQL layer')
```

Comment:
Faudrait être cohérent au niveau du `s`. Soit `Executes and returns`, soit `Execute and return`. À toi de voir mais sinon cool :)